compiler/quilt/__init__.py

```diff
@@ -87,8 +87,10 @@ def handle_interrupt(signum, stack):
     build,
     check,
     config,
+    export,
```

Comment:
please add basic CLI and API docs for quilt.export():
https://github.com/quiltdata/quilt/blob/master/docs/api-python.md
https://github.com/quiltdata/quilt/blob/master/docs/cli.md

note: do not add docs for load() until we resolve the name vs 'import'

Reply:
done